packages/protocol-kit/src/managers/contractManager.ts

```diff
@@ -55,20 +55,23 @@ class ContractManager {
         safeVersion,
         isL1SafeSingleton,
         customSafeAddress: safeAddress,
-        customContracts
+        customContracts,
+        deploymentType: predictedSafe?.safeDeploymentConfig?.deploymentType
```

Comment:
In this `else` this is never possible for any of the contracts. The `if` clause checks if predictedSafe option exists in the config